app/views/home.html

```diff
@@ -1 +1,15 @@
-<h1>Hi this is home page</h1>
+<h1>Pokédex Tracker</h1>
+
+<div class="hero">
+  <img class="mockup" src="mockup.png" width="700px">
+  <p>A tool for tracking your <a href="http://bulbapedia.bulbagarden.net/wiki/Living_Pok%C3%A9dex" target="_blank" class="link">Living Dex</a> progress. This project's source code is open source, available on Github (<a href="https://github.com/robinjoseph08/api.pokedextracker.com" target="_blank" class="link">API</a> &amp; <a href="https://github.com/robinjoseph08/pokedextracker.com/" target="_blank" class="link">website</a>). Feel free to contribute or leave comments or issues!</p>
```

Comment:
i can diggggg it :+1: :+1: three comments:

1. is it weird that we call it the living dex when bulba says living pokedex?
2. should pokémon be capitalized (bulba always capitalizes)
3. my fault, but there should be a comma after "open source" :see_no_evil: